src/settings.ui

```diff
@@ -65,6 +65,13 @@
              </property>
             </widget>
            </item>
+           <item>
+            <widget class="QLabel" name="label_3">
+             <property name="text">
+              <string>(ptz locking depends on PTZ name equal to source camera name)</string>
+             </property>
+            </widget>
+           </item>
```

Comment:
I've merged this hunk into mainline already. It can be dropped from the pull request

Reply:
ok